templates/components/itilobject/timeline/filter_timeline.html.twig

```diff
@@ -81,7 +81,7 @@
    }) %}
 {% endif %}
 
-<div class="filter-timeline float-end position-relative">
+<div class="filter-timeline float-end position-relative ms-auto">
```

Comment:
i guess the float-end class is not needed anymore

Reply:
```ms-auto``` is always needed

Reply:
yes but not `float-end` 😉

Reply:
Oo, sorry !